src/pkg.jl

```diff
@@ -101,13 +101,24 @@ const relative_snoop_locations = (
     joinpath("test", "runtests.jl"),
 )
 
+"""
+    get_snoopfile(pkg_root::String) -> snoopfile.jl
+
+Get's the snoopfile for a package in the path `pkg_root`.
+"""
 function get_snoopfile(pkg_root::String)
     paths = joinpath.(pkg_root, relative_snoop_locations)
     idx = findfirst(isfile, paths)
     idx === nothing && error("No snoopfile or testfile found for package $pkg_root")
     return paths[idx]
 end
 
+"""
+    snoop2root(snoopfile) -> pkg_root
+Given a path to a snoopfile, this function returns the package root.
+If the file isn't of a known format it will return nothing.
+reverse of get_snoopfile(pkg_root)
+"""
```

Comment:
I don't think the implementation matches this.
```
julia> PackageCompiler.snoop2root("package/snoopfile.jl")
"package/"

julia> PackageCompiler.snoop2root("package/snoop/snoopfile.jl")
"package/snoop/"

julia> PackageCompiler.snoop2root("package/test/snoopfile.jl")
"package/test/"

julia> PackageCompiler.snoop2root("package/test/runtests.jl")
"package/"
```
Seems a bit arbitrary so I guess the implementation is off.